avm/res/synapse/workspace/README.md

```diff
@@ -757,6 +757,7 @@ module workspace 'br/public:avm/res/synapse/workspace:<version>' = {
 
 | Parameter | Type | Description |
 | :-- | :-- | :-- |
+| [`accountUrl`](#parameter-accounturl) | string | The account URL of the data lake storage account. |
```

Comment:
Should we bump minor?

Reply:
I usually only bumb minor for breaking changes as we're not 1.0, but as you're asking, sure

Reply:
done

Reply:
Wait, I think until 1.0.0 the intention is to use minor for both minor and major. Then patch for real patch.
What you are suggesting is instead to use minor instead of major, and patch for both minor and patch.
It's quite an important one to agree upon, not just me asking :)

Reply:
That's what I'm saying, yes. Semver does not say anything specific about it because it is pre 1.0.
But at least for this PR, I guess it should be alright now in any case?

Reply:
Approving the PR, but I'll reach out to agree upon the fundamental approach to follow 😗